### DISCUSSION TITLE
Why doesn't sunshine have a more simple way to add/remove programs?

### DISCUSSION THREAD

**Discussion by mamcocob64** [LizardByte/community#255] [Sunshine] [answered]
Why doesn't sunshine have a more simple way to add/remove programs? The current method does not make sense. things like names, commands, outputs... Why? Why can't all these be generated automatically after browsing the windows file manager for the executables or shortcuts? i should not have to add it in manually. if possible i would like to see a natural desktop version that does nothing with the web browser and uses more user friendly methods to make it work. i would burn down nvidia for discontinuing gamestream if i could. i am angry at them

**Comment by ReenigneArcher** [comment, marked as answer]
Sunshine is traditionally for more advanced users. We are slowly improving user friendliness of everything. Not too long ago, Sunshine did not even have a user interface. Anyway, the project is open source and anyone is able to submit PRs to improve things like this.
